script/clone_job.pl

```diff
@@ -113,7 +113,7 @@ =head1 SYNOPSIS
 use JSON;
 
 use FindBin;
-use lib "$FindBin::Bin/../lib";
+BEGIN { unshift @INC, "$FindBin::RealBin/../lib" }
 use OpenQA::Client;
```

Comment:
There is a specific reason why you prefeer to mangle directly with `@INC` and `BEGIN { ... }` instead of `use lib`?

Reply:
@mudler the OBS SR is merged so if you have an updated package installed you can even check yourself. "openqa-clone-job" does not work to resolve OpenQA/Client.pm with the use of `use lib "$FindBin::Bin/../lib";`. But `BEGIN { unshift @INC, "$FindBin::RealBin/../lib" }` as I changed it here works. So I consider it necessary but you are the expert in perl domain :-) Maybe you have a nicer approach with less mangling?

Reply:
Tried with ```use lib "$FindBin::RealBin/../lib";``` instead?

Reply:
that seems to work, updated PR. thx